src/riddle/useCases/solveRiddle/__tests__/createSolveRiddleService.test.ts

```diff
@@ -0,0 +1,126 @@
+
+import { describe, it, expect } from 'vitest';
+import { createSolveRiddleModel } from '../createSolveRiddleService';
+
+const mockRiddle = {
+  id: 'riddle-1',
+  contents: 'What has keys but can’t open locks?',
+  answers: [
+    { id: 'a', text: 'Piano' },
+    { id: 'b', text: 'Map' },
+    { id: 'c', text: 'Locksmith' },
+  ],
+};
+
+describe('createSolveRiddleModel', () => {
+  it('returns true for shouldDisplayRiddle ', () => {
```

Comment:
Strive to formulate test case descriptions from the perspective of the behavior. Eliminate implementation details.

```suggestion
  it('display riddle when answers are available', () => {
```